kufar/books/Dockerfile

```diff
@@ -2,4 +2,9 @@ FROM golang:latest
 WORKDIR /app
 COPY . .
 RUN go build -o main .
+
+FROM alpine:latest
```

Comment:
Please never use `latest` (especially with `alpine` :D ) because it could break your build in future :)
Please pin alpine and golang version

Btw try to avoid using alpine in production ready systems, `debian` and `centos` are more stable, production ready systems.

Reply:
can I use alpine here? Since its not for production, or I need to change it

Reply:
@JustinObanor  You can use it here, but next time please use something stable :)